app/src/main/res/values-ta/strings.xml

```diff
@@ -32,169 +32,166 @@
-    <string name="edit_daily_goal">Edit goal</string>
-    <string name="save_daily_goal">Save</string>
-    <string name="delete_daily_goal">Delete</string>
-    <string name="cancel_daily_goal">Cancel</string>
-    <string name="description_setting_daily_goal">Choose how many sentences you want to record and clips you want to validate (sum of them) per day.</string>
-    <string name="daily_goal_achieved_today">Achieved today</string>
-    <string name="daily_goal_not_achieved_today">Not achieved today</string>
-    <string name="daily_goal_achieved_message">Well done! You achieved your daily goal.\nYou validated {{*{{n_clips}}*}} clip(s) and recorded {{*{{n_sentences}}*}} sentence(s).\nFeel free to continue contributing to Common Voice.</string>
-    <string name="daily_goal_is_not_official_by_mozilla">This daily goal isn\'t an official Mozilla feature. It\'s only a feature of this app.</string>
+    <string name="daily_goal_title">தினசரி இலக்கு</string>
+    <string name="daily_goal_is_not_set">இலக்கு நிர்ணயிக்கவில்லை</string>
+    <string name="set_daily_goal">இலக்கை நிர்ணயம் செய்</string>
+    <string name="edit_daily_goal">இலக்கைத் திருத்து</string>
+    <string name="save_daily_goal">சேமி</string>
+    <string name="delete_daily_goal">நீக்கு</string>
+    <string name="cancel_daily_goal">ரத்துசெய்</string>
+    <string name="description_setting_daily_goal">ஒரு நாளைக்கு நீங்கள் பதிவு செய்ய விரும்பும் வாக்கியங்கள் மற்றும் நீங்கள் சரிபார்க்க விரும்பும் உச்சரிப்புகள் (அவற்றின் தொகை) ஆகியவற்றைத் தேர்வுசெய்க.</string>
+    <string name="daily_goal_achieved_today">இன்று சாதித்தது</string>
+    <string name="daily_goal_not_achieved_today">இன்று ஏதும் சாதிக்கவில்லை</string>
+    <string name="daily_goal_achieved_message">நல்லது! உங்கள் தினசரி இலக்கை நீங்கள் அடைந்துவிட்டீர்கள்.\nநீங்கள் {{*{{n_clips}}*}} உச்சரிப்புகளை சரிபார்த்துள்ளீர்கள் மற்றும் {{*{{n_sentences}}*}} வாக்கியங்களுக்கு உச்சரிப்புகளை பங்களித்துள்ளீர்கள்.\nதCommon Voice-ல் தொடர்ந்து பங்களிக்கவும்.</string>
+    <string name="daily_goal_is_not_official_by_mozilla">இந்த தினசரி இலக்கு அதிகாரப்பூர்வ Mozilla அம்சம் அல்ல. இது இந்த செயலியின் ஒரு அம்சம் மட்டுமே.</string>
     <!-- Message dialog (unspecified category) -->
-    <string name="message_dialog_new_version_available">A new version of the app is available (release {{*{{n_version}}*}}). You can update it from your favourite store (F-Droid, Google Play Store) or directly from GitHub.</string>
-    <!-- Others -->
-    <!-- DO NOT TRANSLATE OR CHANGE THIS - Languages (list) sort "A-Z" - "languages" and "languages_short" sort MUST coincide (e.g. Italian and it, English and en, ect.) -->
-    <!-- DO NOT TRANSLATE OR CHANGE THIS - Abbreviation of Common Voice (e.g. Italian -> it, Swedish -> sv-SE, ect.) -->
+    <string name="message_dialog_new_version_available">செயலியின் புதிய பதிப்பு(வெளியீடு : {{*{{n_version}}*}}) கிடைக்கிறது. நீங்கள் Playstore/F-Droid அல்லது நேரடியாக Github-ல் இருந்து புதுப்பிக்கலாம்.</string>
```

Comment:
Google Play Store is a registered trademark by Google, so do not change it (It's not "Playstore" -> "Google Play Store", better just as "Google Play").

Reply:
Oh. Ok I Will fix.

Reply:
@manimaran96 as I said, please use Crowdin.